js/controllers/form/UserFormHandler.js

```diff
@@ -88,22 +88,20 @@
 		e.preventDefault();
 
 		var $form = this.getHtmlElement(),
-				firstName, lastName, fetchUrl;
-
-		if ($('[name="lastName"]', $form).val() === '') {
-			// No last name entered; cannot suggest. Complain.
-			alert(this.usernameSuggestionTextAlert_);
-			return;
-		}
```

Comment:
What happens when the button is clicked but no name of any kind has been entered?

Reply:
The Username field will be empty and the usual message "This field is required" below that field will be immediately displayed. I think it is OK so, or?

Reply:
:+1: